go/cmd/dolt/commands/commit.go

```diff
@@ -74,32 +63,22 @@ func (cmd CommitCmd) Description() string {
 
 // CreateMarkdown creates a markdown file containing the helptext for the command at the given path
 func (cmd CommitCmd) CreateMarkdown(fs filesys.Filesys, path, commandStr string) error {
-	ap := cmd.createArgParser()
+	ap := actions.CreateCommitArgParser()
```

Comment:
Not a fan of this in the actions package.

Reply:
There was some repeated logic with it in dolt_commit. I felt that it was best to not be repeated. Should that change so that commit.go and dolt_commit.go have their own argparser?

Reply:
I agree that not repeating code is great, but you shouldn't stick it in a random package just because its accessible from both of the packages you are working in.  You can create new packages.

Reply:
Do you think the dolt cli package is the appropriate location? Maybe arg_parsers_helpers.go?